policy/modules/services/aptcacher.if

```diff
@@ -0,0 +1,21 @@
+## <summary>apt-cacher, cache for Debian APT repositories.</summary>
+
+######################################
+## <summary>
+##	read aptcacher config
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to read it.
+##	</summary>
+## </param>
+#
+interface(`aptcacher_read_config',`
+	gen_require(`
+		type aptcacher_etc_t;
+	')
+
+	files_search_etc($1)
+	allow $1 aptcacher_etc_t:dir list_dir_perms;
+	allow $1 aptcacher_etc_t:file mmap_read_file_perms;
+')
```

Comment:
Is this interface used anywhere?

The summary say's "read aptcacher config" but that is not accurate. I would omit the map here, and create another separate `aptcacher_mmap_read_config_files()` (and maybe even yet another `aptcacher_mmap_config_files() to optionally be used with `aptcacher_read_config()`

You should probably also add support that that symlink in there:

```
######################################
## <summary>
##	read aptcacher config
## </summary>
## <param name="domain">
##	<summary>
##	Domain allowed access.
##	</summary>
## </param>
#
interface(`aptcacher_read_config',`
	gen_require(`
		type aptcacher_conf_t;
	')

	files_search_etc($1)
	allow $1 aptcacher_conf_t:dir list_dir_perms;
	allow $1 aptcacher_conf_t:file read_file_perms;
        allow $1 aptcacher_conf_t:file read_lnk_file_perms;
')

######################################
## <summary>
##	mmap aptcacher config files
## </summary>
## <param name="domain">
##	<summary>
##	Domain allowed access.
##	</summary>
## </param>
#
interface(`aptcacher_mmap_config_files',`
	gen_require(`
		type aptcacher_conf_t;
	')

	allow $1 aptcacher_conf_t:file map;
')

######################################
## <summary>
##	mmap read aptcacher config files
## </summary>
## <param name="domain">
##	<summary>
##	Domain allowed access.
##	</summary>
## </param>
#
interface(`aptcacher_mmap_read_config_files',`
	gen_require(`
		type aptcacher_conf_t;
	')

	files_search_etc($1)
	mmap_read_files_pattern(aptcacher_conf_t, aptcacher_conf_t)
')

Reply:
It's not really used so I can just remove the interface I guess